peer/core.yaml

```diff
@@ -335,6 +335,13 @@ ledger:
       # Deploy chaincodes into the genesis block
       chaincodes:
 
+        noop_syscc:
+          path: github.com/hyperledger/fabric/core/system_chaincode/noop
+          type: GOLANG
+          constructor:
+            args:
+
+
```

Comment:
@ghaskins @tamasblummer I like the idea of disabling by default. Couple of other suggestions
  * if we just test for successful return of GetTransactionByUUID (ie, not unmarshal etc) we don't have to worry about failures from unmarshal (such as questions dealing with decryption)
  *  also perhaps not use a global var for ledger (the ledgerH var) but do GetLedger() every time ?